Jenkinsfile

```diff
@@ -180,78 +150,63 @@ pipeline {
             }
             steps{
                 withMaven(maven: 'M3', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LINUX_MVN_RANDOM}') {
-                    retry(3) {
-                        checkout scm
-                    }
                     sh 'mvn javadoc:aggregate -B -DskipStatic=true -DskipTests=true'
                     sh 'mvn deploy -B -T 1C -DskipStatic=true -DskipTests=true -DretryFailedDeploymentCount=10'
                 }
             }
         }
         stage('Quality Analysis') {
-            steps {
-                parallel(
-                    sonarqube: {
-                        node('linux-large') {
-                            retry(3) {
```

Comment:
@mcalcote are we losing the retry functionality by switching away from the scripted syntax? Is there a way we can advise this still?

Reply:
Adding the retry logic back into the top level agent as well as each stage where we use an alternate agent.